dash/backend/src/modules/image/services/image.service.ts

```diff
@@ -28,7 +29,7 @@ export class ImageService {
     ) {
     }
 
-    async getAllImagesByClusterId(clusterId: number): Promise<{total: number, listOfImages: ListOfImagesDto[]}> {
+    async getAllImagesByClusterId(clusterId: number): Promise<{total: string, listOfImages: ListOfImagesDto[]}> {
```

Comment:
Once again, this should probably have the total as a number. Based on a quick check, it would probably make sense to have the DAO cast the count to a number so that it is accurate for everywhere.

With current usages of this it will not make a difference, but if any math is attempted on this total, unexpected behavior is likely with it as a string